misc/openapi-ballerina/modules/ballerina-client-generator/src/main/java/org/ballerinalang/client/generator/CodeGenerator.java

```diff
@@ -40,20 +40,20 @@
 
     /**
      * Generates code(client, server etc) for ballerina source provided  in <code>definitionPath</code>.
-     * <p>Method can be used for generating OAS 3.0.0, Swagger or Ballerina Client</p>
+     * <p>Method can be used for generating OAS 3.0.0, OpenApi or Ballerina Client</p>
```

Comment:
OpenAPI is mentioned twice here by using "OAS 3.0.0" and "OpenAPI"

Reply:
Resolved with 81d9190